kinto/authorization.py

```diff
@@ -122,6 +122,9 @@ def build_permissions_set(object_uri, unbound_permission,
 
     obj_type = get_object_type(object_uri)
 
+    if obj_type is None:
```

Comment:
There is a test for it. I will add a comment. ( https://github.com/mozilla-services/kinto/pull/88/files#diff-78b677143edfdcc21ab2170f88157eacR159 )